pkg/minikube/cni/kindnet.go

```diff
@@ -168,6 +169,12 @@ func (c KindNet) manifest() (assets.CopyableFile, error) {
 
 // Apply enables the CNI
 func (c KindNet) Apply(r Runner) error {
+	// This is mostly applicable to the 'none' driver
+	_, err := r.RunCmd(exec.Command("stat", "/opt/cni/bin/portmap"))
```

Comment:
nit: could we possiblty detect the Runner (if it is not a SSHRunner or kicRunner) then don't do this ? to save micro seconds ?